.readthedocs.yaml

```diff
@@ -9,10 +9,6 @@ build:
   os: ubuntu-20.04
   tools:
     python: "3.9"
-    # You can also specify other tool versions:
```

Comment:
How about moving potential changes to the RTD integration to a separate PR so that the results of several unrelated changes don't create confusion during debugging?